redis/src/main/scala/zio/redis/RedisUri.scala

```diff
@@ -16,8 +16,17 @@
 
 package zio.redis
 
-final case class RedisConfig(host: String, port: Int)
+final case class RedisUri(host: String, port: Int) {
```

Comment:
Please refrain from making unrelated changes. Regarding this particular case, we have already discussed it in the past, so please leave it as it is.

Reply:
I think that bad idea to have in RedisConfig class just host and port variables. First of all i need this structure in RedisClusterConfig and that would be strange imho RedisClusterConfig.config: RedisConfig. All libraries have class for node address.

Reply:
We have already discussed this and agreed not to change it. The fact that it now contains only host and port is very likely temporary. Please do not change things that are already discussed and not related to the scope of a particular pull request.

Reply:
What are u suggesting i need to create RedisUri separate from redis config to use it in RedisClusterConfig or to use inside RedisConfig

Reply:
Returned RedisConfig